.github/workflows/bench.yml

```diff
@@ -14,15 +14,12 @@ jobs:
     runs-on: ubuntu-latest-large
     steps:
       - uses: actions/checkout@v4
-        with:
```

Comment:
since we got rid of zig fastlanez don't think we need this anymore, it's not in ci.yml